apps/js_urls/utils.py

```diff
@@ -0,0 +1,18 @@
+# Python Standard Library
+# Python Standard Library Imports
+import typing as t
```

Comment:
`import typing as T`

Reply:
Removed typing to make it compatible with Python 2